README.md

```diff
@@ -168,6 +168,8 @@ command="unlock" to your authorized_keys before the key, e.g.
 
     command="unlock" ssh-rsa .....
 
+> In some cases (CentOS stream) you may wish to replace `unlock` with `systemd-tty-ask-password-agent`
+
```

Comment:
How is this CentOS specific? Wouldn't any systemd distro have access to that executable?

Reply:
In my case I don't think I had the `unlock` command

Reply:
I think more generic wording is definitely appropriate here. Something like
> If your distribution provides `systemd-tty-ask-password-agent` or another alternative to manage unlocking of volumes needed to boot, you may prefer to restrict access to that program instead of `unlock`.

is preferable. We should be reducing distribution-specific instructions in the README, not increasing it. At some point I intend to rewrite the entire installation section to reduce or eliminate the Red Hat content. There's no reason those instructions should be any more detailed than for other distributions in the list. In addition, because none of the currently active maintainers are Red Hat users or contributors, packaging for CentOS will probably fall by the wayside with subsequent releases unless somebody wants to pick up unofficial packaging (and documentation) duties for that operating system. We want to get out of the distribution packaging game in this repo.